### ISSUE TITLE
Cover art image is needlessly duplicated in albom directory

### ISSUE THREAD

**Issue by ronso0** [mixxxdj/mixxx#11966]
I have an album directory with these files
000_album-name.jpg
101_track_1.mp3
102_track_2.mp3
...

When I have `102_track_2.mp3` loaded in a deck, clear the cover and re-select `000_album-name.jpg` Mixxx creates
`102_track_2.jpg` (which is identical to `000_album-name.jpg`).

Why?
This seems entirely unnecessary.

Was it ever like this or was it added by #10902 ?

**Comment by daschuer** [comment]
I agree. Just from the top of my head, this should be caused by the "copy to track folder" feature. 
Mixxx tries to put the cover into a place where it can be found without database access. 
000_album-name.jpg is not such a location. 
You may try to rename the cover to cover.jpg to avoid that. 

To fix the bug, I think we need to extend the possible cover locations, that your case is covered.

**Comment by ronso0** [comment]
> To fix the bug, I think we need to extend the possible cover locations, that your case is covered.

The image file name includes the album artist and the album title, `000 black sun empire - driving insane-***.jpg`, using the same pattern as the track files, `101 black sun empire - [title].mp3`.

Maybe it's also possible to detect whether the source image is already
* in the same directory as the track file and
* the directory only contains tracks with the same album title

otherwise copy the image there.

**Comment by ronso0** [comment]
Related: #12299
